mkosi.build

```diff
@@ -90,3 +90,8 @@ cat > "$DESTDIR"/etc/issue <<EOF
 Kernel \r on an \m (\l)
 
 EOF
+
+# Manually update the boot loader from the one we just built
+mkdir -p "$DESTDIR"/boot/efi/EFI/systemd "$DESTDIR"/boot/efi/EFI/BOOT
+cp "$DESTDIR"/usr/lib/systemd/boot/efi/systemd-bootx64.efi "$DESTDIR"/boot/efi/EFI/systemd/systemd-bootx64.efi
+cp "$DESTDIR"/usr/lib/systemd/boot/efi/systemd-bootx64.efi "$DESTDIR"/boot/efi/EFI/BOOT/bootx64.efi
```

Comment:
Hmm, are those paths (with /boot/efi) always correct?

Reply:
in the mkosi image we build they are...